README.md

```diff
@@ -0,0 +1,32 @@
+# Anyscale Starter Templates
+
+These templates are a set of minimal examples & tutorial for customers to run on the anyscale platform.
+
+## Contributing Guide
+
+If the template is generic to Ray & Ray libraries, please consider adding the template in: https://github.com/ray-project/ray/tree/master/doc/source/templates
+
+
+To add a template:
+
+1. Add your template as a directory under `templates/<your-template-name>`
+
+    For example:
+
+    ```text
+    templates/my-awesome-template
+        <name-of-your-template>/
+            README.md
+            <name-of-your-template>.ipynb/py
+    ```
+
+    Your template does not need to be a Jupyter notebook. It can also be presented as a
+    Python script.
+
+    All templates MUST have a `README.md` file.
+
+2. Add your compute configuration under `configs/<your-template-name>` (for both AWS and GCE).
```

Comment:
Can I also add "cluster environment"?

This is necessary for most of our content.

Reply:
You can add a BYOD, meaning build your own docker image, publish it and users can use it.

The reason we are not supporting cluster envs now is for the build time, we want users to be able to click no a template and start it right away. It's a little bit more work for now on the template authoring to build the image and publish it, but that's to get the best experience on the user side. If you need help publishing docker images, I am happy to help.